### DISCUSSION TITLE
SQL基礎語法資源分享

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS3073-C/Python-Community-GS3073-C#471] [General]
透過這個影片讓我在程式設計的專題中，對SQL的基礎語法運用自如，收穫很大，算是講得非常易懂對新手很友善的，分享給大家。

https://www.youtube.com/watch?v=gvRXjsrpCHw

**Comment by brian431** [comment]
> 透過這個影片讓我在程式設計的專題中，對SQL的基礎語法運用自如，收穫很大，算是講得非常易懂對新手很友善的，分享給大家。
> 
> 
> 
> [https://www.youtube.com/watch?v=gvRXjsrpCHw](url)

點進去不會通往那個連結，可能是因為用到url?連結直接複製貼上即可。

**Reply by manipulateed** [reply]
了解~~感謝幫忙

**Comment by williamyen0623** [comment]
SQL的應用非常廣泛，值得學習，感謝分享！

**Comment by gnsJhenJie** [comment]
如果只是單機版的程式，也可以試試看SQLite，幾乎所有語法和SQL都一樣，就不用另外架設MySQL或是MariaDB的伺服器了。

**Comment by lx3211** [comment]
如果需要應用資料庫的話，SQL語法是很重要的基礎

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by limitx0** [comment]
謝謝分享，我也想要來學 SQL

**Comment by Ben6uo** [comment]
看來很快就會到人人會用SQL的日子了
